pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -112,6 +113,20 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 		return errors.Wrap(err, "failed to create csi config map")
 	}
 
+	// update the msgr2 flag
+	for _, m := range cluster.ClusterInfo.Monitors {
+		// taking last 4 character will be vaild for both IPv4 Aand IPv6 address
+		// m.Endpoint=10.1.115.104:3300
+		monPort := string(m.Endpoint[len(m.Endpoint)-4:])
```

Comment:
We may want to use a lib or regex to extract the port. Especially for external mode, t's possible that someone could be using a nonstandard mon port like 12345 that isn't 4 chars long. Or do we require standard ports for external clusters? If there is a nonstandard port, how do we determine if it's a v2 port?

```suggestion
		monPort := string(m.Endpoint[len(m.Endpoint)-4:])
```